GetStream/stream-chat-android
# [i196] support auto-translation feature in Compose

### 🎯 Goal

Related ticket: https://github.com/GetStream/android-internal-board/issues/196

Related PR:
- https://github.com/GetStream/stream-chat-android/pull/5020
- https://github.com/GetStream/stream-chat-android/pull/5022
- https://github.com/GetStream/stream-chat-android/pull/5023

### 🎨 UI Changes

#### Compose

| Before | After |
| --- | --- |
| ![Before](https://github.com/GetStream/stream-chat-android/assets/1286516/7dfb1a70-f13e-4572-912a-978c190e64d4) | ![After](https://github.com/GetStream/stream-chat-android/assets/1286516/ecd67b45-c086-43ff-b093-c3503ea817fc) | 


### 🧪 Testing

#### Test Steps:
1. Login as `Kanat`
2. Open p2p channel with `Tommaso`
3. Make sure you can see `Translated to French`

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_
